modules/packages/Sort.chpl

```diff
@@ -796,6 +796,111 @@ module BinaryInsertionSort {
   }
 }
 
+pragma "no doc" 
+module TimSort {
+  private use Sort;
+
+    /*
+    Sort the 1D array `Data` using a parallel tim sort algorithm.
```

Comment:
I think it's just 1 word:

```suggestion
    Sort the 1D array `Data` using a parallel timsort algorithm.
```

at least according to the original impl: https://github.com/python/cpython/blob/master/Objects/listsort.txt

Reply:
UPD:Fixed